systemd/systemd.go

```diff
@@ -405,16 +406,45 @@ func (l Log) String() string {
 	return fmt.Sprintf("%s %s %s", l.Timestamp(), l.SID(), l.Message())
 }
 
+// UseFuse detects if we should be using squashfuse instead
```

Comment:
The method also shouldn't be public (no reason to call it from outside), and it needs some tests covering its functionality, otherwise we'll break it and won't notice. Should be trivial by mocking the resources depended on (we need a test covering each) and verifying that the service file generated looks good.